proto/pb/contextBox.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.8
+// 	protoc        v3.21.5
```

Comment:
We should probably use the same versions when generating the proto files as we keep overwritting these

Reply:
Good shout, let me have a look ho we can enforce it.

Reply:
Do you think this could help us enforce it?

```sh
# Enforce same version of protoc 
PROTOC_VERSION = "3.21.8"
CURRENT_VERSION = $$(protoc --version | awk '{print $$2}')
# Generate all .proto files 
proto:
	@if [ "$(CURRENT_VERSION)" = "$(PROTOC_VERSION)" ]; then \
		protoc --go-grpc_out=. --go_out=. proto/*.proto ;\
	else \
		echo "Please update your protoc version. Current $(CURRENT_VERSION) | Required $(PROTOC_VERSION)"; \
	fi
```

Reply:
should work 👍